src/_articles/archive/zones.md

```diff
@@ -152,33 +154,51 @@ Code might execute in other nested child zones as well,
 but at a minimum it always runs in the root zone.
 
 
-## Handling asynchronous errors
+## Handling uncaught errors
 
 One of the most used features of zones is
-their ability to handle uncaught errors
-in asynchronously executing code.
-The concept is similar to a try-catch in synchronous code.
-
-An _uncaught error_ is often caused by some code using `throw`
-to raise an exception that is not handled by a `catch` statement.
-Another way to produce an uncaught error is
-to call `new Future.error()` or
-a Completer's `completeError()` method.
-
-Use the `onError` argument to `runZoned()` to
-install a _zoned error handler_—an asynchronous error handler
-that's invoked for every uncaught error in the zone.
-For example:
+their ability to catch and handle uncaught errors,
+both synchronous and asynchronous.
+An _uncaught error_ is often caused by code using `throw`
+to raise an exception without an accompanying `catch`
+statement to handle it.
+
+In `async` functions, errors usually end up as the result of a Future.
+Futures allow you to handle errors where you `await` the future.
+Sometimes an error escapes, or a future is not `await`-ed,
+in which case an error becomes an uncaught error.
+Uncaught errors can crash your program.
+
+Zones can install an _uncaught error handler_ upon creation.
+When an uncaught error occurs in that zone,
+rather than just crashing the program,
+it calls its uncaught error handler.
+(In fact, any zone that doesn't specify a handler
+inherits the uncaught error handler of the root zone,
+which just crashes the program on an uncaught error).
+
+The simplest way to introduce a new zone with an uncaught error handler
+is to use `runZoneGuarded`. Its `onError` callback becomes the
+uncaught error handler of a new zone, and handles any
+synchronous errors thrown by the call as well.
 
 <!-- run_zoned1.dart -->
 {% prettify dart tag=pre+code %}
-runZoned(() {
+runZonedGuarded(() {
   Timer.run(() { throw 'Would normally kill the program'; });
-}, onError: (error, stackTrace) {
+}, (error, stackTrace) {
   print('Uncaught error: $error');
 });
 {% endprettify %}
 
+_See also [`Zone.fork`][], [`Zone.runGuarded`][]
+and [`ZoneSpecification.uncaughtErrorHandler`][]
+for more uncaught error handling methods in zones._
```

Comment:
**issue:** Why, then how.
```suggestion
To learn more about uncaught error handling methods in zones,
see [`Zone.fork`][], [`Zone.runGuarded`][]
and [`ZoneSpecification.uncaughtErrorHandler`][]
```

**nit:** I really dislike `For more...`. It lacks agency.

Reply:
These are all the other ways to handle uncaught errors with zones, so not just "To learn more about..." but.... idk how to phrase it otherwise.

Maybe one of these?
> You can handle uncaught errors using other zone APIs, including ...

or

> Other methods of handling uncaught errors with zones are ...

Reply:
**reply:** OK, that's a bit different. I like the first one or maybe:
```markdown
Zones can also handle uncaught errors using...
```